forum/qa-plugin/user-activity-log/user-activity-log-search.php

```diff
@@ -192,8 +196,8 @@ private function findUsersID(string $user)
 
     private function findUsersPostsLinks(string $event,string $params)
     {
-        $first = substr($event, 0, 1);
-        if($first != 'q' && $first != 'a' && $first != 'c'){
+        $first = substr($event, 0, 2);
+        if($first != 'q_' && $first != 'a_' && $first != 'c_'){
```

Comment:
Czy nie prościej jest umieścić te litery w tablicy i na raz sprawdzić czy tablica nie zawiera wartości ze zmiennej?

Nie wiem czy użycie [`in_array`](https://www.php.net/manual/en/function.in-array.php) jest tu ok, ale np.:
```php
$unwantedEventPrefixes = ['q_', 'a_', 'c_'];

if (!in_array($eventPrefix, $unwatedEventPrefixes)) {
```